### ISSUE TITLE
Inner class 'Song' of ComboBoxView not found

### ISSUE THREAD

**Issue by paulroemer** [vaadin/vaadin-combo-box-flow#22]
Class is instantiated via
```
DemoView demoView = clazz.newInstance();
demoView.setParameter(null,"");
com.vaadin.flow.dom.Element javaDemo = demoView.getElement();
javaDemo.setAttribute("slot", "java-demo");
getElement().appendChild(javaDemo);
```

This causes
```
java.lang.BootstrapMethodError: java.lang.NoClassDefFoundError: com/vaadin/ui/combobox/ComboBoxView$Song
	at com.vaadin.ui.combobox.ComboBoxView.createObjectComboBox(ComboBoxView.java:129)
	at com.vaadin.ui.combobox.ComboBoxView.initView(ComboBoxView.java:95)
	at com.vaadin.flow.demo.DemoView.<init>(DemoView.java:68)
	at com.vaadin.ui.combobox.ComboBoxView.<init>(ComboBoxView.java:33)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at com.vaadin.site.view.elements.ElementDetailsView.lambda$onLocationChange$0(ElementDetailsView.java:132)
```

Maybe I hit a Java limitation I do not know? I can't imagine the artifact is broken.

**Comment by paulroemer** [comment]
On the other hand even IntelliJ's decompiler does not show the inner class `Song`.

**Comment by asashour** [comment]
I guess this should be fixed as stated in #24
